stdlib/base.pkl

```diff
@@ -1901,9 +1984,22 @@ class Mapping<out Key, out Value> extends Object {
 
   /// Folds the entries of this mapping in iteration order using [operator], starting with [initial].
   external function fold<Result>(initial: Result, operator: (Result, Key, Value) -> Result): Result
+  
+  /// Tells if [predicate] holds for every entry of this mapping.
+  /// 
+  /// Returns [true] for an empty mapping.
+  external function every(predicate: (Key, Value) -> Boolean): Boolean
+  
+  /// Tells if [predicate] holds for at least one entry of this mapping.
+  /// 
+  /// Returns [false] for an empty mapping.
+  external function any(predicate: (Key, Value) -> Boolean): Boolean
 
   /// Converts this mapping to a [Map].
   external function toMap(): Map<Key, Value>
+  
+  /// Converts this mapping to a [Dynamic] object.
+  external function toDynamic(): Dynamic
```

Comment:
How about documenting that late binding is not preserved?
"Returns a Dynamic that delegates to this mapping" would be a clear explanation of a good implementation.

Reply:
> "Returns a Dynamic that delegates to this mapping" would be a clear explanation of a good implementation.

This would be the only case where we document _implementation_ as opposed to _outcome_ in the standard library.